dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -407,4 +418,28 @@ protected void patch(Context context, HttpServletRequest request, String apiCate
         throw new RepositoryMethodNotImplementedException("No implementation found; Method not allowed!", "");
     }
 
+    protected void updateMetadata(DSpaceObjectService dsoService,
```

Comment:
please add javadoc here

Reply:
N/A - method dropped in favor of MetadataConverter#setMetadata in latest rebase